torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -915,6 +915,10 @@ void runNondiffOptimization(
   for (const auto& passPair : getCustomPostPasses()) {
     passPair.first(graph);
   }
+  if(tensorExprFuserEnabled()){
```

Comment:
This is in the legacy executor, we don't need to run `RemoveTensorTypeSpecializations` here because the types are guaranteed by the inputs, since we cache and specialize. I've been trying to delete it but unfortunately there's a legacy glow integration internally :/